index.html

```diff
@@ -5,8 +5,31 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Sam - Portofolio</title>
+    <script src="https://kit.fontawesome.com/b39c945343.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="assets/style.css" />
 </head>
 <body>
-    
+    <header id="header_menu">
+        <a class="welcome_text">
+            Welcome
+        </a>
+        <a href="#" class="toggleNav">
+            <img src="assets/icons/union.png" alt="union icon" />
+        </a>
```

Comment:
- [ ] Kindly wrap the hamburger button (mobile menu button) in a `nav` tag instead of the `a` tag for semantic reasons 👍